src/commons/unipoolTokenDistributorHandler.ts

```diff
@@ -72,19 +83,11 @@ export function onWithdrawn(params: OnStakedParams): void {
   const userAddress = params.userAddress;
   const distributor = params.distributor;
   createUnipoolContractInfoIfNotExists(contractAddress);
-  updateLastUpdateDate(contractAddress);
-  updateTotalSupply(contractAddress);
-  updateRewardPerTokenStored(contractAddress);
-  userWithdrew(userAddress, amount, distributor);
-}
 
-export function onRewardUpdated(contractAddress: Address, userAddress: string, contractName: string): void {
-  createUnipoolContractInfoIfNotExists(contractAddress);
-  updateRewardRate(contractAddress);
```

Comment:
@aminlatifi 
You have removed `updateRewardRate()` but it's not in `updateReward` function, is it ok?